corehq/apps/users/views/utils.py

```diff
@@ -185,3 +186,21 @@ def get_locations_with_orphaned_cases(domain, location_ids, user_id):
         loc.get_path_display(): case_count_per_loc[loc.location_id] for loc in all_locs
         if loc.location_id in case_count_per_loc
     }
+
+
+def get_user_location_info(domain, user_location_ids, user_id):
+    shared_locations = _get_location_ids_with_other_users(
+        domain,
+        user_location_ids,
+        user_id
+    )
+
+    orphaned_case_count_per_location = _get_locations_with_orphaned_cases(
```

Comment:
Looks like `_get_locations_with_orphaned_cases` also  calculates `_get_location_ids_with_other_users` inside it. Is it worth quickcaching?

Reply:
Good point

Reply:
https://github.com/dimagi/commcare-hq/pull/33184/commits/b1626985200778532e8c63e8d44f5af95e9c5770